components/record/EntityField.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <li
+    v-if="locale !== null"
+    :lang="locale"
```

Comment:
The conditional for whether or not displaying the lang attribute introduces a lot of new duplication of code. I am wondering if it would work if you use `:lang="locale ? locale : false "` instead.

Reply:
right, https://vuejs.org/v2/guide/syntax.html#Attributes
It should be fine already just setting it to whatever is in the code/locale key, since I'm setting it to null when it's not required and only to an empty string for undefined